qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -216,6 +218,7 @@ def search(self, text, *, ignore_case='never', reverse=False,
     def clear(self):
         self.search_displayed = False
         self._widget.findText('')
+        self._tab.scripts.update_stylesheet(searching=False)
```

Comment:
This broke some tests, as we call `.clear()` in some places before the JavaScript code is initialized. I changed it so this only gets called when `search_displayed` really is `True`: 19589c93b5da244b98e3af57c633d02291c98a3a